uint/src/uint.rs

```diff
@@ -1632,3 +1633,26 @@ macro_rules! impl_quickcheck_arbitrary_for_uint {
 macro_rules! impl_quickcheck_arbitrary_for_uint {
 	($uint: ty, $n_bytes: tt) => {};
 }
+
+
+#[cfg(feature = "arbitrary")]
+#[macro_export]
+#[doc(hidden)]
+macro_rules! impl_arbitrary_for_uint {
+	($uint: ty, $n_bytes: tt) => {
+		impl $crate::arbitrary::Arbitrary for $uint {
+			fn arbitrary(u: &mut $crate::arbitrary::Unstructured<'_>) -> $crate::arbitrary::Result<Self> {
+				let mut res = [0u8; $n_bytes];
+				u.fill_buffer(&mut res)?;
+				Ok(Self::from(res))
```

Comment:
```suggestion
				let mut res = Self::zero();
				u.fill_buffer(&mut res.0)?;
				Ok(res)
```

Reply:
Unfortunately the `unstructured.fill_buffer(&bytes)` will only take `u8` and so this won't work as the word sizes are `u64`.

Reply:
Ah, right, good point.